doc/lua_api.txt

```diff
@@ -2434,8 +2434,11 @@ These functions return the leftover itemstack.
     * Optional: Variable number of arguments that are passed to `func`
 
 ### Server
-* `minetest.request_shutdown([message],[reconnect])`: request for server shutdown. Will display `message` to clients,
-    and `reconnect` == true displays a reconnect button.
+* `minetest.request_shutdown([message],[reconnect],[delay])`: request for server shutdown. Will display `message` to clients,
+    `reconnect` == true displays a reconnect button,
+    `delay` permit to add an optionnal delay before shutdown
+        negative delay cancel the current active shutdown
```

Comment:
Would prefer `minetest.cancel_shutdown_requests` for nicer code.

A more engineered solution would have ids like in the hud api so that a mod only cancels their own requests, but that's over engineered I feel.